libs/components/forms/src/lib/modules/checkbox/checkbox-group.component.ts

```diff
@@ -112,4 +126,5 @@ export class SkyCheckboxGroupComponent {
   readonly #idSvc = inject(SkyIdService);
   protected errorId = this.#idSvc.generateId();
   protected formErrorsDataId = 'checkbox-group-form-errors';
+  protected headingClass = 'sky-font-heading-3';
```

Comment:
Is this the default value? should we call that out in the jsdocs

Reply:
the default is actually 3, the class gets constructed from the input. I'll add it in the jsDocs